api-tests/dev_apis/crypto/test_c072/test_c072.c

```diff
@@ -132,8 +132,7 @@ int32_t psa_pake_set_peer_test(caller_security_t caller __UNUSED)
       /* Expect a bad state when psa_pake_set_peer is called on aborted inactive operation object */
         status = val->crypto_function(VAL_CRYPTO_PAKE_SET_PEER,
                                        &operation,
```

Comment:
To maintain consistency with other test cases, check for bad state can be added as
```
if( check1[i].expected_status == PSA_SUCCESS)
{
    status = val->crypto_function(VAL_CRYPTO_PAKE_SET_PEER, &operation, check1[I].peer_id, check1[I].peer_id_len);                                           
}